src/ert/_c_wrappers/job_queue/workflow_job.py

```diff
@@ -48,6 +52,14 @@ class WorkflowJob:
 
     def __post_init__(self):
         self.__running = False
+        self._ert_script: Optional[type] = None
+        if self.script is not None and self.internal:
+            try:
+                self._ert_script = ErtScript.loadScriptFromFile(self.script)  # type: ignore
+            except Exception as err:
```

Comment:
this is a very wide exception type to except - can we narrow it down?

Reply:
`loadScriptFromFile` it self only raises `ValueError`, maybe we can catch that one and have an "unexpected error ... " for the catch-all class afterwards?

Reply:
Well the script can throw any exception it wants and we will have a hard time controlling it. We want any failure to load to result in this message.

Reply:
fair enough.